bin/sync_all

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-cd $(readlink --canonicalize --no-newline ${0%/*})/..
```

Comment:
dang it, you are right. that was an error.  I am sorry, I am new to this git thing.  What should I do?